types/types.cc

```diff
@@ -118,7 +118,7 @@ sstring boolean_to_string(const bool b) {
 sstring inet_addr_type_impl::to_sstring(const seastar::net::inet_address& addr) {
     std::ostringstream out;
     out << addr;
-    return out.str();
+    return std::move(out).str();
```

Comment:
i'd suggest drop this helper and switch to `fmt::to_string(addr)`.

Reply:
Fixed. This helper was not needed and used only in two places. Moreover, for `seastar::net::inet_address` there is a formatter specialization :+1: